packages/destination-actions/src/destinations/cordial/index.ts

```diff
@@ -7,7 +7,7 @@ import addContactToList from './addContactToList'
 import removeContactFromList from './removeContactFromList'
 
 const destination: DestinationDefinition<Settings> = {
-  name: 'Cordial',
+  name: 'Actions Cordial',
```

Comment:
@dlasky is this where name is supposed to be defined? I see slack has just name `Slack` but in app it is `Slack (Actions)`

https://github.com/segmentio/action-destinations/blob/main/packages/destination-actions/src/destinations/slack/index.ts#L6

If defined here I would suggest 
```suggestion
  name: 'Cordial (Actions)',
```

Reply:
yeah i'm not sure if we settled on that 100% for a convention (aside from disambiguation from classics) but seems reasonable to me.